### PR TITLE
[WIP] Initial structure for Android SDK operator

### PR DESCRIPTION
**Pull request by dimitraz**

### CODE REVIEW

**Comment by dimitraz** [comment]
ping @maleck13 @philbrookes @philipgough @witmicko

File: pkg/androidSdk/complete.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	"github.com/Sirupsen/logrus"
+	api "github.com/aerogear/android-sdk-operator-poc/pkg/apis/androidsdk/v1"
+	"github.com/operator-framework/operator-sdk/pkg/sdk"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// deletePod cleans up suceeded pods and sets the resource's status to "Done"
+// once complete
+func deletePod(h *Handler, helper *api.AndroidSDK, podName string) error {
```
Comment (dimitraz) [inline pkg/androidSdk/complete.go:15]: need to change `helper` to a better name, but didn't want to collide with `sdk`. 
open to suggestions :)

File: deploy/cr.yaml [hunk 2]
```diff
@@ -1,4 +1,8 @@
 apiVersion: "androidsdk.aerogear.org/v1"
 kind: "AndroidSDK"
 metadata:
-  name: "example"
+  name: "android-sdk-pkg-config"
+spec:
+  ConfigMapName: android-sdk-config
+status: 
+  Phase: Install
```
Comment (maleck13) [inline deploy/cr.yaml]: The Phase should probably be empty. My understanding is that the status is controlled by the operator not the user

Comment (dimitraz) [reply deploy/cr.yaml]: :+1:

**Pull request merged**